src/term/mod.rs

```diff
@@ -950,12 +954,16 @@ impl ansi::Handler for Term {
 
     #[inline]
     fn save_cursor_position(&mut self) {
-        err_println!("[unimplemented] save_cursor_position");
+        self.saved_cursor = Some(self.cursor);
     }
 
     #[inline]
     fn restore_cursor_position(&mut self) {
-        err_println!("[unimplemented] restore_cursor_position");
+        if let Some(pos) = self.saved_cursor {
+            self.cursor = pos;
```

Comment:
The terminal doesn't crash or anything when this happens, but it's not handled gracefully either. Does alacritty implement `SIGWINCH` yet?